src/services/userService.js

```diff
@@ -0,0 +1,50 @@
+let users = [];
+let userId = 1;
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getAllusers = () => {
```

Comment:
Consider renaming `getAllusers` to `getAllUsers` to follow standard naming conventions and ensure consistency.